### PR TITLE
Switching off duplicated timestamp

### PR DESCRIPTION
**Pull request by acierto**
Signed-off-by: bnechyporenko <bnechyporenko@bol.com>

## Hey, I just made a Pull Request!

Suggesting to remove duplicating timestamp.

Screenshot 2022-09-30 at 22 38 12

One timestamp comes from task step itself, and second from configured `winston`. 
It's just a noise, should be only 1 timestamp on the line.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by freben — APPROVED** [review]
Haha, this led me down an interesting path through basically the whole code base. I was about to suggest keeping the timestamp inside the entry metadata for completeness, and just switching out the `simple()` logger for one that knows not to print the timestamp specifically. But then I backtracked, since it doesn't really provide much value since the timestamp is available anyway so easily to any consumer. And besides the extra timestamp wastes space in every log message.

So to summarize, let's ship it! Thanks again.

**Comment by suuus** [comment]
Thank you for contributing during Hacktoberfest 🍂! You can claim the Backstage Hacktoberfest Holopin at https://bck.st/hacktoberfest-holopin 🙌🏻
